packages/cli-kit/src/haiku.test.ts

```diff
@@ -0,0 +1,27 @@
+import {inTemporaryDirectory, write} from './file.js'
+import {join} from './path.js'
+import {generate} from './haiku.js'
+import {describe, expect, test, vi} from 'vitest'
+
+describe('generate', () => {
+  test('rerolls the name if a directory exists with the same name', async () => {
```

Comment:
We could add a test in this file to check that the generated value follows the expected pattern.

Reply:
Makes sense, I'll move the check from the prompt test.